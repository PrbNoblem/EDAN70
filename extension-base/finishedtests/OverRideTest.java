package finishedtests;

public class OverRideTest extends SuperClassOR {

	public int m1(int x) {
		return 3;
	}



	public void m2() {
	}

	@Override
	public void m3() {

	}

}
