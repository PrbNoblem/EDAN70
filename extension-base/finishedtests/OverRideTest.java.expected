package finishedtests;

public class OverRideTest extends SuperClassOR {

	public int m1(int x) {
		return 3;
	}

	@Override
	public void m2() {
	}

	@Override
	public void m3() {

	}

}
